pkg/encryption/path_test.go

```diff
@@ -63,3 +66,97 @@ func forAllCiphers(test func(cipher storj.CipherSuite)) {
 		test(cipher)
 	}
 }
+
+func TestSegmentEncoding(t *testing.T) {
```

Comment:
Let's test:
* encoding/decoding random data paths,
* parsing completely random data
* parsing all byte slices up to length 3. (or if it takes too long, then 2)

Reply:
done